Update durable-functions-eternal-orchestrations.md

brandonh-msft opened the pull request:
`ContinueAsNew` requires a parameter; I'm guessing this is supposed to be `(null)` here.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

brandonh-msft committed:
Update durable-functions-eternal-orchestrations.md

`ContinueAsNew` requires a parameter; I'm guessing this is supposed to be `(null)` here.

cgillum reviewed (approved):
Good catch, thanks!

#sign-off
